h2o-py/tests/testdir_algos/xgboost/pyunit_feature_interactions.py

```diff
@@ -16,7 +16,14 @@ def xgboost_feature_interactions():
     feature_interactions = model.feature_interaction(2, 100, -1)
     assert isinstance(feature_interactions[0], H2OTwoDimTable)
     assert len(feature_interactions) == 11
-    
+
+    tempdir = "./"
```

Comment:
usually for storing test outputs we use `path = pyunit_utils.locate("results")`
also I would call locate once not again on every line

Reply:
e9696f4